config/routes.rb

```diff
@@ -2,9 +2,11 @@
 
 Rails.application.routes.draw do
   # Pages
+  get '', to: 'pages#thankyou'
```

Comment:
```suggestion
```

Change line 14 instead of this (commented line above it to remember how to activate the normal homepage)

```ruby
# root to: 'pages#index'

root to: 'pages#thankyou'
```